src/Codeception/Util/Locator.php

```diff
@@ -33,27 +51,22 @@ class Locator
      * use \Codeception\Util\Locator;
      *
      * $I->fillField(Locator::combine('form input[type=text]','//form/textarea[2]'), 'qwerty');
-     * ?>
      * ```
      *
      * As a result the Locator will produce a mixed XPath value that will be used in fillField action.
      *
      * @static
-     *
      * @param $selector1
      * @param $selector2
-     *
-     * @throws \Exception
-     *
-     * @return string
+     * @throws Exception
      */
-    public static function combine($selector1, $selector2)
+    public static function combine($selector1, $selector2): string
```

Comment:
Parameters are here for documentation purpse only, because function uses `func_get_args`, but they both must be strings.